order.go

```diff
@@ -86,6 +86,7 @@ func (c *Client) CaptureOrder(orderID string, captureOrderRequest CaptureOrderRe
 	if err != nil {
 		return capture, err
 	}
+	req.Header.Set("Prefer", "return=representation") // Verbose response: https://developer.paypal.com/docs/api/orders/v2/#orders-authorize-header-parameters
```

Comment:
could we rather make this an option on the client?

as in after you do `NewClient` could we do

```
c.SetReturnRepresentation()
```